go.mod

```diff
@@ -26,12 +26,7 @@ require (
 	github.com/zgalor/weberr v0.6.0
 	gitlab.com/c0b/go-ordered-json v0.0.0-20171130231205-49bbdab258c2
 	gopkg.in/square/go-jose.v2 v2.6.0
-)
-
-require (
-	github.com/go-logr/logr v1.2.3 // indirect
```

Comment:
These three aren't really being deleted, but combined with the other indirect dependencies below.

`k8s.io/apimachinery` is being upgraded as an explicit dependency that was previously misplaced below.